### DISCUSSION TITLE
Change the date format independently of the language settings

### DISCUSSION THREAD

**Discussion by Orionnss** [stoatchat/discussions#602] [Feature Suggestions]
Hello,

Currently the only way to change date format is to switch language.
I'm french and used to the DD/MM/YYYY format, but I want my UI to be in english, because it's easier to communicate instructions and stuff to other people.

In US English, it's MM/DD/YYYY format.

Fortunately, there is the traditional english language settings that allows me to get the better of both worlds, but since I intend to learn new languages and use UI in other languages in the future, I would like to be able to choose the date format in the settings, so I can keep use the UI in my preferred language and be as efficient as before to understanding when a message has been posted and prevent me from doing mistakes.

Thank you and sorry if it has already been posted :)

(It seems that other post are more community-centered settings, but I would like it as a personal setting)

**Comment by ThomasJRyan** [comment]
Seems to be implemented as of now
